app/Http/Controllers/User/OvertimeController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Http\Controllers\User;
+
+use Illuminate\Http\Request;
+use App\Models\Overtime;
+use Auth;
+use Validator;
+use App\Http\Controllers\Controller;
+
+class OvertimeController extends Controller
+{
+    public function index()
+    {
+        $overtime = Overtime::where('user_id', Auth::user()->id)->paginate(config('app.pagination'));
+        return view("employees.overtime.index", ['overtime' => $overtime]);
+    }
+
+    public function create()
+    {
+        return view("employees.overtime.create");
+    }
+
+    public function store(Request $request)
+    {
+        $overtime = new Overtime();
+        $overtime->date = $request->date;
+        $overtime->start_time = $request->from;
+        $overtime->end_time = $request->to;
+        $overtime->content = $request->content;
+        // count hours OT
+        $toTime = strtotime($overtime->end_time);
```

Comment:
@nguyenvanpho dùng các function của `Carbon` để tín toán liên quan đến ngày tháng

Reply:
@damnnkst  anh ơi phần này là lấy ngày giờ nhập từ bàn phím bên view sang sau đó tính toán nên em không dùng function của Carbon để tính toán được ạ.